api/nfd/v1alpha1/types.go

```diff
@@ -141,12 +149,17 @@ type NodeFeatureGroup struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
-	Spec   NodeFeatureGroupSpec   `json:"spec"`
+	// Spec defines the rules to be evaluated.
+	Spec NodeFeatureGroupSpec `json:"spec"`
+
+	// Status of the node feature group after the most recent evaluation of the
```

Comment:
```suggestion
	// Status of the nodeFeatureGroup after the most recent evaluation of the
```
no?

Reply:
I just spelled out the natural language version of the purpose of the object 🙂 But I don't have strong opinions. Changed as suggested.

Reply:
for node feature we have NodeFeature, so is just to have a similar form